release.yaml

```diff
@@ -105,9 +105,9 @@ harvester-csi-driver:
 system-upgrade-controller:
 - 101.0.0+up0.3.3
 ui-plugin-operator:
-- 101.0.0+up0.1.0
+- 101.0.0+up0.1.0-rc3
 ui-plugin-operator-crd:
-- 101.0.0+up0.1.0
+- 101.0.0+up0.1.0-rc3
```

Comment:
We are moving backwards from 0.1.0 to 0.1.0-rc3?

Reply:
We’re not moving backwards; previous updates of the chart never modified the rc version in the Chart.yaml itself